vendor.mod

```diff
@@ -6,6 +6,10 @@ module github.com/docker/cli
 
 go 1.24.0
 
+replace github.com/moby/moby/client => github.com/moby/moby/client v0.0.0-20251216111441-25fbe6cd43eb
+
+replace github.com/moby/moby/api => github.com/moby/moby/api v0.0.0-20251216111441-25fbe6cd43eb
```

Comment:
Are replace rules needed? Wouldn't vendoring the pseudo-version with the newest commit work?